src/services/auth-service/auth-service.ts

```diff
@@ -86,15 +89,25 @@ export class AuthService {
     }
 
     public getUserDetails = (): Promise<User> => {
+        const reqId = uuid();
+        this.actions.progressFn(reqId, true);
         return this.apiClient
             .get<UserDetailsResponse>('/users/current')
-            .then(resp => ({
-                email: resp.data.email,
-                firstName: resp.data.first_name,
-                lastName: resp.data.last_name,
-                uuid: resp.data.uuid,
-                ownerUuid: resp.data.owner_uuid
-            }));
+            .then(resp => {
+                this.actions.progressFn(reqId, false);
+                return {
```

Comment:
UserService could be used to obtain user data with correct naming already applied.

Reply:
Could you explain more ? reqId must be unique per request if you refer to this